src/plone/restapi/services/site/get.py

```diff
@@ -38,6 +39,7 @@ def __call__(self, expand=False):
                 "plone.site_logo": site_settings.site_logo and getSiteLogo() or None,
                 "plone.robots_txt": site_settings.robots_txt,
                 "plone.allowed_sizes": image_settings.allowed_sizes,
+                "plone.site_timezone": default_timezone(self.context),
```

Comment:
I would say that we should get this setting from registry and not from the `default_timezone` function.

According to [its implementation](https://github.com/plone/plone.app.event/blob/master/plone/app/event/base.py#L449-L455) the function is designed to return the correct timezone even per-user, so if we call this endpoint with different users, we could get different results.

I would say that if we add the site_timezone setting, this should be the one set by the `plone.portal_timezone` setting.

I am OK on returning also the users own timezone, but I would return that in a different setting and do not call it "site_timezone" but "user_timezone".

Reply:
Renaming the key does not help. The implementation ambiguously returns either the user or site timezone. It would be better to name the key as `plone.timezone` because it is totally ambiguous from this implementation.

Thus I would go with @erral's suggestion. Get the value from the registry. However, I would return the user's timezone from the user endpoint, not the site endpoint.

Reply:
Ok, I agree, changed as required